services/QuillLMS/db/structure.sql

```diff
@@ -452,6 +452,47 @@ CREATE SEQUENCE public.activity_classifications_id_seq
 ALTER SEQUENCE public.activity_classifications_id_seq OWNED BY public.activity_classifications.id;
 
 
+--
+-- Name: activity_healths; Type: TABLE; Schema: public; Owner: -
+--
+
+CREATE TABLE public.activity_healths (
```

Comment:
These were not related to this PR, but were not versioned before. I checked with Cissy that it's ok to include these structure.sql changes in this PR.